src/bundles/SellerRegistration/components/DocumentsForm/DocumentsForm.js

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Form, actions } from 'react-redux-form';
+
+import 'whatwg-fetch';
+
+//import { required } from '../../../../validators';
+
+import Layout        from '../../../../shared/Layout';
+import BaseForm      from '../../../../shared/form/BaseForm';
+import SubmitForm    from '../../../../shared/form/SubmitForm';
+import ErrorBox      from '../../../../shared/form/ErrorBox';
+import StatefulError from '../../../../shared/form/StatefulError';
+
+import formProps     from '../../../../shared/reduxModules/formPropsSelector';
+
+class DocumentsForm extends BaseForm {
+
+  static propTypes = {
+    action: React.PropTypes.string,
+    csrf_token: React.PropTypes.string,
+    form: React.PropTypes.object.isRequired,
+    dispatch: React.PropTypes.func.isRequired,
+    model: React.PropTypes.string.isRequired
+  }
+
+  state = {
```

Comment:
state should be set up in the constructor? this seems like an anti-pattern to me